docs/developer-docs/daos/nns/proposal-voting.mdx

```diff
@@ -0,0 +1,174 @@
+It might be useful to share the following information with others, so that followers can make well-informed decisions who to follow for which topic:
+* **Who you are**: Name of the person or group and their relationship to the ICP. You may want to argue why the people behind the neuron have a long-term interest in the wellbeing of the ICP and are therefore motivated to make good voting decisions.
+* **Neuron control**: How controls the neuron? Does it have a single controller or can it only be changed by a group of people?
+* **Covered topics**: A description of what topics the known neuron will vote on. Does the neuron commit to always vote on these topics?
+Since following is done based on proposal topics, this is relevant information for followers.
+Usually, neurons would want to follow other neurons neurons that always vote on a given topic in order not to miss out on [voting rewards](./staking-voting-rewards.mdx).
+* **Depth of reviews**: A description of how the known neuron plans to verify proposals. To which depth will it review the code or the content of the proposal and how is this done?
+* **Following setting**: For the topics where the know neuron does not vote directly, who does the neuron follow?
+* **Voting principles**: What are the principles according to which you plan to vote.
+* **Link to the neuron ID / proposal**: If you already created the neuron, you may want to provide its ID or even link to a dashboard where it can be inspected. The same holds for the proposal to register it as a know neuron in the case where the proposal was already made.
+
+:::info
+There are no rules how a know neuron has to vote. However, due to the above reasons, it might be useful for known neurons to **focus on fewer topics and commit to always vote on those** rather than trying to cover a lot of topics and vote less reliably. For example, they could become experts on some proposal topics. If a known neuron commits to always vote on a certain topic it might be more likely that others follow the neuron on this topic as the followers want to make sure they are not missing out on rewards.
+
+:::
+
+
+### Register the known neuron
+To register a neuron as a know neuron, you need to submit a NNS proposal. The proposal can be submitted by any neuron that has at
+least 10 ICP staked.
+For simplicity, we assume in the following that the neuron to be registered was created on the NNS dapp and that the same neuron is
+used to submit the proposal.
+
+**1. Get DFX**
+It is not possible to submit proposals through the NNS Dapp. Therefore, use a command line tool called DFX.
+Unfortunately, DFX is only natively supported on Mac and Linux, but it can be installed on Windows as well by following the [instructions for installing DFX on all platforms](https://internetcomputer.org/docs/current/developer-docs/getting-started/install/#installing-dfx).
+
+Here a copy of the instructions for Mac and Linux for simplicity:
```

Comment:
```suggestion
**1. Download and install `dfx`**
It is not possible to submit proposals through the NNS dapp. Therefore, you must use a command line tool called `dfx`.`dfx` is only natively supported on Mac and Linux, but it can be installed on Windows using WSL. 

To install `dfx`, follow the [installation instructions](/docs/current/developer-docs/getting-started/install/#installing-dfx).

Here are the instructions for Mac and Linux:
```